### PR TITLE
Add on off to display

### PR DESCRIPTION
**Pull request by blavka**
and code refactoring

### CODE REVIEW

File: bcl/src/bc_module_lcd.c [hunk 1]
```diff
@@ -155,13 +165,14 @@ uint32_t bc_module_lcd_draw_char(uint16_t left, uint16_t top, uint8_t ch)
     return w;
 }
 
-void bc_module_lcd_draw_string(uint16_t left, uint16_t top, char *str)
+uint16_t bc_module_lcd_draw_string(uint16_t left, uint16_t top, char *str)
```
Comment (hubpav) [inline bcl/src/bc_module_lcd.c]: Why uint16_t and not int?

File: bcl/src/bc_module_lcd.c [hunk 2]
```diff
@@ -7,13 +7,20 @@
 
 #include <bc_module_lcd.h>
 
+#define BC_MODULE_LCD_DISP_ON   0x04
```
Comment (hubpav) [inline bcl/src/bc_module_lcd.c]: These defines should begin with underscore since they are private

Comment (blavka) [inline bcl/src/bc_module_lcd.c]: ok

File: bcl/src/bc_module_lcd.c (hunk 1 above)
Comment (blavka) [inline bcl/src/bc_module_lcd.c]: Ok, change to int

**Pull request merged**